volume/drivertest/drivertest.go

```diff
@@ -0,0 +1,230 @@
+package drivertest
```

Comment:
you need an extra linebreak after the boilerplate because godeps will assume that this huge comment applies to the package